.github/workflows/tests.yml

```diff
@@ -72,8 +72,12 @@ jobs:
           php_version: 7.3
           composer_version: 1
 
-      - name: Pest
-        run: ./vendor/bin/pest --coverage --coverage-clover='coverage.xml'
+      - name: Install WP Tests
+        run: |
+          sudo /etc/init.d/mysql start && ./bin/install-wp-tests.sh wordpress_test root root localhost latest
+
+      - name: PHPUnit
+        run: ./vendor/bin/phpunit --coverage-clover='coverage.xml'
```

Comment:
We might need to add `XDEBUG_MODE=coverage` here to enable code coverage, see https://github.com/sebastianbergmann/php-code-coverage/issues/834#issuecomment-736337016.